mirror.php

```diff
@@ -117,15 +131,15 @@ public function syncV2()
             throw new \UnexpectedValueException('Cannot save last timestamp to last_metadata_timestamp in '.getcwd().'. Make sure the file is writable.');
         }
         $lastTime = trim(file_get_contents($this->getTimestampStorePath()));
-
+        $this->output('Last update was on '.new DateTimeExt($lastTime).PHP_EOL);
         $changesResp = $this->client->request('GET', $this->apiUrl.'/metadata/changes.json?since='.$lastTime, ['headers' => ['Host' => parse_url($this->apiUrl, PHP_URL_HOST)]]);
         if ($changesResp->getHeaders()['content-encoding'][0] !== 'gzip') {
             throw new \Exception('Expected gzip encoded responses, something is off');
         }
         $changes = json_decode($changesResp->getContent(), true);
 
         if ([] === $changes['actions']) {
-            $this->output('No work' . PHP_EOL);
+            $this->output('No changes since '. new DateTimeExt($lastTime) . PHP_EOL);
```

Comment:
the DateTimeExt class is rather unnecessary IMO. You can use `date(DATE_ATOM, $lastTime)` here or similar.

Reply:
Got it, it's true.